usaon_vta_survey/templates/response/relationships/application_societal_benefit_area.html

```diff
@@ -68,12 +67,11 @@ <h4>Societal Benefit Areas</h4>
 
     <h4>Relationship</h4>
     {% if sqla_inspect(relationship).transient %}
-      {{ application_societal_benefit_area_fields(form.relationship.form) }}
+      {{ render_form(form.relationship.form) }}
```

Comment:
Wondering if there is a way to alter the submit button to say "create new relationship"

Reply:
Does `button_map` support it?

Reply:
Looked at the docs, and no, `button_map` is only for styles. I think this needs to be set in the `SubmitField` object? Either way, I think we should open an issue for this and move on :)